.github/dependabot.yml

```diff
@@ -1,10 +1,10 @@
 version: 2
 updates:
-  - package-ecosystem: "bundler"
+  - package-ecosystem: "cargo"
     directory: "/"
     schedule:
-      interval: "daily"
+      interval: "weekly"
```

Comment:
Feel free to change these to monthly, if weekly is too often. We've changed a number of other repos to monthly recently. Manual Dependabot runs can be triggered via the GitHub UI now, which combined with monthly can give the best of both worlds.